Common/DTO/UserAvatar.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DTO
+{
+    public class UserAvatar
+    {
+        public Guid Id { get; set; }
```

Comment:
Why do we have 2 id here ? We will save id of avatar on the client?

Reply:
No, we create this object only in UserService. Second Id needed to search for user avatar in the database

Reply:
I think we don't need Id in UserAvatar in DTO